### ISSUE TITLE
Localize search input placeholder

### ISSUE THREAD

**Issue by bugzilla-to-github** [mozilla/pontoon#2182]
*This issue was created automatically by a [script](https://github.com/mathjazz/bugzilla2github/).*
## [Bug 1586234](https://bugzilla.mozilla.org/show_bug.cgi?id=1586234)
Bug Reporter: @adngdb
CC:   @mathjazz, monikamaheshwari1996@gmail.com
See also:     https://bugzilla.mozilla.org/show_bug.cgi?id=1586233

The search input placeholder in the search module needs to be localized.

This is going to be difficult because the content of that placeholder is dynamic, based on the various statuses and filters that are selected in the FiltersPanel component. This might collide with bug 1586233.

**Comment by Mitch4sho** [comment]
Hi @mathjazz, it's been a while since we last spoke! 😄 I was hoping to take on more tasks for Pontoon next quarter. I came across this P5 issue and am interested in working on it. This would be an excellent way to reintroduce myself to the codebase and get back into things. Please let me know your thoughts, and I look forward to hearing from you!

**Comment by Mitch4sho** [comment]
Yes, I would be interested in taking that ticket. Where can I find the setup documentation for Pontoon again?

**Comment by mathjazz** [comment]
Let's continue in #2298. :)
